app/models/user.rb

```diff
@@ -405,9 +405,6 @@ def child_without_permission?
     child? && UserParent.where( "user_id = ? AND donorbox_donor_id IS NULL", id ).exists?
   end
 
-  EMAIL_CONFIRMATION_RELEASE_DATE = Date.parse( "2022-12-14" )
-  EMAIL_CONFIRMATION_REQUIREMENT_DATETIME = DateTime.parse( "2025-01-15 12:00" )
-
```

Comment:
If you're intent is to remove email confirmation requirements entirely, you need to audit the code to remove all references to these constants. Some of your spec failures are legit and related to this.